src/styles/_catalog.scss

```diff
@@ -1,6 +1,19 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: $card-width;
+
+  justify-content: center;
+  gap: 48px;
+
+  @media (min-width: 448px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
 
   max-width: 944px;
   margin: 0 auto;
```

Comment:
I think it's unnecessary because you use justify-content: center;

Reply:
you can't add margin for BEM block, remove it